### ISSUE TITLE
Should tag names be included in style definitions?

### ISSUE THREAD

**Issue by ericyd** [tylerchilds/cutestrap#35]
I really like the minimalism of Cutestrap, but I have found myself fighting with the button styling when trying to use semantic markup in my nav elements.  I was recently reading through [BEM naming conventions](http://getbem.com/naming/) and noticed that it specifies "No tag name or ids" for CSS selectors.  Conversely, [Cutestrap docs](https://www.cutestrap.com/) state that "A default button is applied with the .btn class or automatically on button tags or button/submit inputs."

Is this implementation of tag styling in Cutestrap intentional? Am I referencing incorrect documentation for BEM?

**Comment by tylerchilds** [comment]
Good point. I think this might be a conflict of interest between providing a default button style and and BEM. I took a lot of liberty with styling many tags, like blockquote. Buttons are more of a gray area, is it a standard UI element or is it a BEM block? I think it's a little bit of both, which is why I chose `.btn` and for `<button>`.

I think it would be a good idea to drop the as opinionated state of the default tags, similar to how the other default form elements require classes to operate fully.
